src/Modules/Finance/Header/Total/Total.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import styles from '../Form.module.css';
+
+interface ITotalProps {
+	value: number
+}
+
+export const Total: React.FC<ITotalProps> = props => {

```

Comment:
Wrap in `React.memo` as well.
https://reactjs.org/docs/react-api.html#reactmemo

Reply:
Done